distribution/docs/pom.xml

```diff
@@ -190,6 +190,7 @@
             <plugin>
                 <groupId>br.com.ingenieux</groupId>
                 <artifactId>jbake-maven-plugin</artifactId>
+                <version>0.0.9</version>
```

Comment:
we _could_ move this version declaration to the top level pom with the others, for consistency, but this also works.

Reply:
Will move